constants/addresses.ts

```diff
@@ -1,13 +1,20 @@
 export const LENDING_POOL = "0x7d2768dE32b0b80b7a3454c06BdAc94A69DDc7A9";
 export const INCENTIVES_CONTROLLER =
```

Comment:
we could add the suffix `_MAINNET` to distinguish the two addresses

Reply:
I was planning to add it in the next commit, because I'll have to update imports on the tests too 👍🏻